src/Tuura/Concept/Circuit/Derived.hs

```diff
@@ -199,3 +201,14 @@ internals ints = interfaceConcept $ \s ->
                  if s `elem` ints
                  then Internal
                  else Unused
+
+combinationalGate :: Eq a => Expr a -> a -> CircuitConcept a
```

Comment:
Btw, you can also write it as follows:

`complexGate (Var a `And` Var b) (Not (Var a) `And` Not (Var b)) c`

This is a bit more readable.